content/docs/user-guide/assets/pipeline/asset-cache.md

```diff
@@ -0,0 +1,44 @@
+---
+linkTitle: Asset Cache 
+title: Asset Cache
+description: In Open 3D Engine (O3DE), the Asset Cache stores the runtime optimized product assets for your project, as well as process job and dependency information.
+weight: 600
+toc: true
+---
+
+The **Asset Cache** stores the runtime optimized product assets for each target platform, information about the process jobs that generated the product assets, and additional information about the assets such as product dependencies. The Asset Cache is located in the `Cache` directory under your project's root directory.
+
+{{< important >}}
+Don't add the Asset Cache to source control. Because the Asset Cache is continuously updated through automatic processes, it's difficult to manage within source control. Using an Asset Cache from source control can also cause issues with asset timestamps that are used to determine whether assets need to be reprocessed.
+
+Don't edit product assets directly. If **Asset Processor** reprocesses the source asset, any changes made to the product asset are overwritten. To make changes to an asset, edit an upstream file, the source asset file, or the `.assetinfo` sidecar file.
+{{< /important >}}
+
+## Asset Cache contents
+
+There are three parts to the Asset Cache: the **Asset Database**, the **Asset Catalog**, and the **Asset Cache** itself.
+
+
+### Asset Database
+
+The Asset Database is only used by Asset Processor. The Asset Database is a development time database containing details about assets, jobs, **Asset Builders**, and scan directories. It contains all the information Asset Processor requires to determine which assets have been processed, and which assets need to be processed.
+
+With the information in the Asset Database, Asset Processor can be suspended in the middle of a process job and resume when relaunched, remembering all the pending process jobs in the job queue.
+
+### Asset Catalog
+
+The Asset Catalog is a runtime catalog of product assets and product dependencies. The Asset Catalog contains a subset of information from the Asset Database; the information about assets required at runtime. **Asset Browser** uses the Asset Catalog to find the runtime product assets on disk that you request for your projects. 

```

Comment:
```suggestion
The Asset Catalog is a runtime catalog of product assets and product dependencies. The Asset Catalog contains a subset of information from the Asset Database: the information about assets required at runtime. **Asset Browser** uses the Asset Catalog to find the runtime product assets on disk that you request for your projects.
```

Reply:
I think the semi colon is correct but I'm going to change this in a new commit.

Reply:
Yeah, not a big deal for me. But I just thought that you generally had to have a complete sentence following a semicolon (unless it's used in a list).